var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -240,6 +242,7 @@ class Cp2k(MakefilePackage, CMakePackage, CudaPackage, ROCmPackage):
 
         with when("@2024.2:"):
             depends_on("dla-future-fortran@0.1.0:")
+            depends_on("dla-future-fortran@0.2.0:", when="@master")
```

Comment:
Is it already known what the next release of CP2K will be called? If yes, might be worth just saying
```suggestion
            depends_on("dla-future-fortran@0.2.0:", when="@1234.5:")
```
? If no, never mind, but I guess this will need to be updated once that's known.

Reply:
Good point, thanks. The next release will be `2025.1`, so I change it to that.